e2e/cypress/e2e/next-internal/tags.cy.ts

```diff
@@ -44,6 +44,7 @@ context('Tags editing', () => {
 
     cy.wait('@getTags');
 
+    cy.wait(500);
```

Comment:
Isn't waiting `500ms` after waiting for `@getTags` too much?

Reply:
I'll try less